src/Test.UnitTests.Sarif/TestData/InsertOptionalDataVisitor/ExpectedOutputs/CoreTests-Absolute_TextFiles.sarif

```diff
@@ -65,7 +65,7 @@
       "artifacts": [
         {
           "location": {
-            "uri": "file:///REPLACED_AT_TEST_RUNTIME/src/Test.UnitTests.Sarif/TestData/InsertOptionalDataVisitor/InsertOptionalDataVisitor.txt"
+            "uri": "file:///ENLISTMENT_ROOT/src/Test.UnitTests.Sarif/TestData/InsertOptionalDataVisitor/InsertOptionalDataVisitor.txt"
```

Comment:
>ENLISTMENT_ROOT [](start = 28, length = 15)

Why did you change this name? It was named previously as a clue that this value is rewritten. We've lost that information now.

Or did you discover that the renaming no longer occurs?

Reply:
`ENVIRONMENT_DEPENDENT_ENLISTMENT_ROOT`?

---
In reply to: [475662994](https://github.com/microsoft/sarif-sdk/pull/2035#discussion_r475662994) [](ancestors = 475662994)

Reply:
That doesn't indicate to readers that the test runtime replaces this value. Can you please just restore it to its previous value?

---
In reply to: [475669236](https://github.com/microsoft/sarif-sdk/pull/2035#discussion_r475669236) [](ancestors = 475669236,475662994)